README.md

```diff
@@ -3,154 +3,110 @@ This repository holds the kcp tests that tests against the publicly available  i
 * golint installed. See [Installing golint](https://github.com/golang/lint#installation)
-* Have the environment variable `KUBECONFIG` set pointing to your cluster
-
-### Include new test folder
-If you create a new folder for your test case, **add the path** to the [include.go](https://github.com/openshift/openshift-tests-private/blob/master/test/extended/include.go)
-
-### Create go-bindata for new YAML files
-If you have some **new YAML files** used in your code, you have to generate the bindata first.
-Run `make update` to update the bindata. For example, you can see the bindata has been updated after running the `make update` as follows:
-```console
-$ git status
-	modified:   test/extended/testdata/bindata.go
-	new file:   test/extended/testdata/olm/etcd-subscription-manual.yaml
-```
-
-### Compile the executable binary
-Note that we use the `go module` for package management, the previous `go path` is deprecated.
-```console
-$ git clone git@github.com:openshift/openshift-tests-private.git
-$ cd openshift-tests-private/
-$ make build
-mkdir -p "bin"
-export GO111MODULE="on" && export GOFLAGS="" && go build -o "bin" "./cmd/extended-platform-tests"
-$ ls -hl ./bin/extended-platform-tests 
--rwxrwxr-x. 1 cloud-user cloud-user 165M Jun 24 22:17 ./bin/extended-platform-tests
-```
-
+* Install [kubelogin](https://github.com/int128/kubelogin.git) and [KCP kubectl plugin](https://github.com/kcp-dev/kcp.git) ( Ensure your installed KCP kubectl plugin version is the same with the kcp service version )
+  - If you would like to download the suitable version packages directly go to [kcp release assets](https://github.com/kcp-dev/kcp/releases), download the zip file suitable for your OS, extract and copy the binaries to a place which is in your PATH and once copied run `kubectl-kcp --version` to verify if server matches with client 
+  - If you would like to build the plugin by your self you could follow the steps like below:
+  ```
+  $ git clone https://github.com/kcp-dev/kcp.git
+  # E.g. Install the release-0.5 version KCP kubectl plugin
+  # commitID: "fccfb61" of release-0.5 branch/tag
+  $ git checkout fccfb61
+  $ make install WHAT=./cmd/kubectl-kcp && make install WHAT=./cmd/kubectl-workspaces && make install WHAT=./cmd/kubectl-ws
+  ```
+
+* Have the environment variable `KUBECONFIG` set pointing to your kcp service
+* Login the kcp service via SSO (Single Sign On)
+  ```
+  kubectl oidc-login get-token --oidc-issuer-url=https://sso.xxxx.com/auth
+   --oidc-client-id=cloud-services --oidc-redirect-url-hostname=127.0.0.1
+  ```
+* If you would like to test BYO cases please make sure to set the `PCLUSTER_KUBECONFIG` environment variable else all the tests related to BYO will be skipped. 
 ## Contribution 
 Below are the general steps for submitting a PR to master branch. First, you should **Fork** this repo to your own Github account.
 ```console
```

Comment:
Though seems not have impact, why console is used? I see many occurrences.

Reply:
Hmm, it's origin fomat I don't change it, I think we could safely remove the `console`.

Reply:
Ah, sorry, now I figured it out. No need to remove. With it, the non-command words will appear with different color.

Reply:
Good to know.

Reply:
Sorry, with my last comment above, I mean keeping `console` can make non-command lines appear with good blue color like:
```console
$ make build
mkdir -p "bin"
export GO111MODULE="on" && export GOFLAGS="" && go build  -ldflags="-s -w" -mod=mod -o "bin" "./cmd/kcp-tests"
```
Now I see your https://github.com/Phaow/kcp-tests/blob/readme-enhancements/README.md kept other places of `console` and removed some places of `console`, e.g.:
```
$ make build
mkdir -p "bin"
export GO111MODULE="on" && export GOFLAGS="" && go build  -ldflags="-s -w" -mod=mod -o "bin" "./cmd/kcp-tests"
```
This may confuse newcomers to execute both mkdir and export. Could you add console back for those lines? Sorry if caused trouble.

Reply:
Hmm, I just replace `console` to `shell` for CLI execute items, only the log output keep `console`.
Use shell
![image](https://user-images.githubusercontent.com/86950592/191170330-311f383c-b191-4095-bf60-52e728575e6e.png)

Use console
![image](https://user-images.githubusercontent.com/86950592/191170442-86889a9e-cac5-40fb-9fc2-4c3a2e3cafaf.png)

Seems shell looks better for CLI execute. WDYT? OR we change all to `console`?

Reply:
Seems my local `md` plugin is not the same display with github. I changed all to `console` and remove the comment `#` for enhancing seems looks better now.